asapdiscovery-docking/asapdiscovery/docking/workflows/large_scale_docking.py

```diff
@@ -189,20 +201,20 @@ def check_inputs(cls, values):
                 "Must specify postera_molset_name if uploading to postera."
             )
 
-        if fragalysis_dir and structure_dir:
-            raise ValueError("Cannot specify both fragalysis_dir and structure_dir.")
-
-        if not fragalysis_dir and not structure_dir:
-            raise ValueError("Must specify either fragalysis_dir or structure_dir.")
+        # fragalysis dir, structure dir and PDB file are mutually exclusive
```

Comment:
do we need more checks to make sure that exactly one of these 3 has been specified?

Reply:
Done now